### PR TITLE
bugfix in ljspeech/tts2

### PR DESCRIPTION
**Pull request by beckgom**
in stage 4, train_config should be used, not decode_config.

### CODE REVIEW

**Comment by kan-bayashi** [comment]
No. `decode_config` should be used.

**Comment by kan-bayashi** [comment]
Sorry. I misunderstood.
You are right, but `ourdir` should keep as is.

File: egs/ljspeech/tts2/run.sh [hunk 1]
```diff
@@ -182,13 +182,13 @@ if [ ${stage} -le 4 ] && [ ${stop_stage} -ge 4 ]; then
...
            --valid-json ${dt_json} \
-           --config ${decode_config}
+           --config ${train_config}
 fi
 
 if [ ${n_average} -gt 0 ]; then
     model=model.last${n_average}.avg.best
 fi
-outdir=${expdir}/outputs_${model}_$(basename ${decode_config%.*})
+outdir=${expdir}/outputs_${model}_$(basename ${train_config%.*})
```
Comment (kan-bayashi) [inline egs/ljspeech/tts2/run.sh:191]: `outdir` should be derived from `decode_config`.
`expdir` comes from `train_config`.

```suggestion
outdir=${expdir}/outputs_${model}_$(basename ${decode_config%.*})
```

**Comment by kan-bayashi** [comment]
Thanks!

**Comment by beckgom** [comment]
> Sorry. I misunderstood.
> You are right, but `ourdir` should keep as is.

Oh. Yes. you are absolutely right. my bad :0

**Pull request merged**